src/il_representations/scripts/joint_training.py

```diff
@@ -418,11 +559,27 @@ def bc_setup(venv, obs_encoder, n_batches, shuffle_buffer_size,
     return bc_learner, bc_aug_fn, il_demo_webdatasets
 
 
+@gail_ingredient.capture
+def gail_setup(venv, obs_encoder, n_batches, shuffle_buffer_size,
```

Comment:
It seems quite a few parameters in this function definition are not used. Can they be removed?

Reply:
Deleted function & other GAIL joint train stuff